codacy/values-production.yaml

```diff
@@ -66,22 +66,22 @@ codacy-ingress:
   create: true
   ingress:
     hostname:
-      app: <--- codacy-app.dns.internal ---> # Codacy application DNS 
-      api: <--- codacy-api.dns.internal ---> # Codacy API DNS
+      app: <--- codacy-app.dns.internal ---> # Codacy application hostname
+      api: <--- codacy-api.dns.internal ---> # Codacy API hostname
     annotations:
       kubernetes.io/ingress.class: nginx
       kubernetes.io/tls-acme: "true"
       certmanager.k8s.io/acme-challenge-type: http01
       certmanager.k8s.io/cluster-issuer: letsencrypt
```

Comment:
This one depends on us having https://github.com/jetstack/cert-manager installed in the cluster.

This component tries to automaticallly generate and validate a certificate for you.  Given that we don't mention it anywhere else, I think its better for us to leave this line out for now